test/de/opendiabetes/tests/plugin/importer/TestImporterUtil.java

```diff
@@ -9,10 +9,10 @@
 
 @Ignore
 public class TestImporterUtil {
-    public static Importer getImporterFromPath(String path){
-        PluginManager manager = new DefaultPluginManager();
-        manager.loadPlugin(Paths.get(path));
-        manager.startPlugins();
+    public static Importer getImporter(String importer){
+        PluginManager manager = new DefaultPluginManager(Paths.get("export"));
```

Comment:
this loads all plugins in export by default, thus makes testing plugins one by one harder, as many plugins get loaded extending the same Importer interface. I would prefer having both methods in Util, but using the old one for testing

Reply:
@ocastx, review please :)

Reply:
this issue seems to be fixed with the build script modifications, added assertions to test if only one plugin is loaded with the new version in
https://github.com/lucasbuschlinger/BachelorPraktikum/pull/15/commits/44f107390e90ea60338b88130522d4a06bc6ef52

Reply:
So what now?
Is it fine or do you  want to revert to how it was?

Reply:
Its fine ;)